main.py

```diff
@@ -154,17 +282,14 @@ def main(cl_arguments):
     if EMAIL_NOTIFIER:
         EMAIL_NOTIFIER(body="Starting run.", prefix="")
 
-    _try_logging_git_info()
+    _log_git_info()
 
     log.info("Parsed args: \n%s", args)
 
     config_file = os.path.join(args.run_dir, "params.conf")
     config.write_params(args, config_file)
     log.info("Saved config to %s", config_file)
 
-    seed = random.randint(1, 10000) if args.random_seed < 0 else args.random_seed
-    random.seed(seed)
-    torch.manual_seed(seed)
     log.info("Using random seed %d", seed)
```

Comment:
how does this not error out? Seed seems to be no longer defined here and not a global?

Reply:
Oh whoops that's weird - seems like Git didn't update all the changes here

Reply:
figuring out what's going on